crypto/openssl/openssl_key_exchange_ecdh.c

```diff
@@ -220,29 +207,29 @@ static bool compute_publicBECDH(ecdh_context_t *key_ex_data)
 	char *hexbuf1 = BN_bn2hex(x);
 
 	LOG(LOG_DEBUG, "Bx %s : bytes %d, %s\n",
-	    BN_is_negative(x) ? "Negative" : "Positive", bn_num_bytes(x),
-	    hexbuf1);
+			BN_is_negative(x) ? "Negative" : "Positive", bn_num_bytes(x),
+			hexbuf1);
 	OPENSSL_free(hexbuf1);
 
 	char *hexbuf2 = BN_bn2hex(y);
 
 	LOG(LOG_DEBUG, "By %s : bytes %d, %s\n",
-	    BN_is_negative(y) ? "Negative" : "Positive", bn_num_bytes(y),
-	    hexbuf2);
+			BN_is_negative(y) ? "Negative" : "Positive", bn_num_bytes(y),
+			hexbuf2);
 	OPENSSL_free(hexbuf2);
```

Comment:
Printing value may reveal info to the attacker

Reply:
Done.